src/mgr/ActivePyModules.cc

```diff
@@ -526,15 +526,15 @@ PyObject *ActivePyModules::get_typed_config(
   const std::string &module_name,
   const std::string &key) const
 {
-  if (!py_module_registry.module_exists(module_name)) {
-    derr << "Module '" << module_name << "' is not available" << dendl;
-    Py_RETURN_NONE;
-  }
-
   std::string value;
   bool found = get_config(module_name, key, &value);
   if (found) {
-    PyModuleRef module = py_module_registry.get_module(module_name);
+    PyModuleRef module = py_module_registry.find_module(module_name);
+    if (module == nullptr) {
```

Comment:
`nullptr` is converted to `PyModuleRef` before comparison. Just use shared_ptr bool operator instead:
```
if (!module) {
 ...
}
```